src/components/hooks/useLockBodyScroll.tsx

```diff
@@ -1,6 +1,6 @@
 import React, {useLayoutEffect} from 'react'
 
-function useLockBodyScroll(props) {
+function useLockBodyScroll(props:string) {
```

Comment:
`props` doesn't describe the parameter correctly. I suggest something like `action`

Reply:
Done ✅